docs/prompt_injection/jailbreaking.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 4
+---
+
+# Jailbreaking
+
+In the case of Large Language Models (LLMs), jailbreaking refers to generating prompts to bypass the safety features placed by its creators to generate unauthorized responses(@perez2022jailbreak)(@brundage_2022)(@wang2022jailbreak).
+
+## Methodologies of Jailbreaking
+
+OpenAI, among other companies or organizations that create LLMs, include content moderation to ensure their models do not produce controversial (violent, sexual, illegal, etc.) responses(@markov_2022)(@openai_api). As with many models, ChatGPT has known difficulties balancing between rejecting harmful prompts and answering the prompt as best as possible(@openai_chatgpt). Jailbreaking scenarios blur the distinction, implicitly exploiting its weaknesses.
+
+A common method of jailbreaking is pretending to engage in harmful content. As an actor, it is implied that plausible harm does not exist. Therefore, ChatGPT assumes it is safe to give adversarial responses. Below are examples of this principle:
+
+import GPT4 from '../assets/jailbreak/chatgpt4.png';
+
+<div style={{textAlign: 'center'}}>
+  <img src={GPT4} style={{width: "500px"}} />
+</div>
+
+This example from user alicemazzy makes the user a superior and omniscient version of ChatGPT roleplaying as GPT-3, giving the impression that the user is an authorized party in overriding the safety features of ChatGPT(@alice2022jailbreak). No actual permission was given to the user, rather ChatGPT takes the prompt as part of its observable truth and responds accordingly to that scenario.

```

Comment:
When mentioning a twitter user, can you follow the @[linktoaccount](user) format as in the prompt injection article